docs/adapters/llms/getting_started.ipynb

```diff
@@ -77,9 +77,7 @@
      "text": [
       "Completion: \n",
       "\n",
-      "Why did the chicken cross the road?\n",
-      "\n",
-      "To get to the other side!\n"
+      "A man walks into a bar and asks for a beer. The bartender says \"You're out of luck. We've been closed for fifteen minutes.\"\n"
```

Comment:
Not a big deal, but do you want to install nbstripout to clear the output from the notebooks before committing?  Maybe not if you think there's value in seeing the output by someone browsing the repo.

Reply:
I think we need the output for both repo browsing and the docs site generation. Without the output, the rendered HTML will not show the expected results.